python-librarian-sdr/Config.in

```diff
@@ -0,0 +1,8 @@
+config BR2_PACKAGE_PYTHON_LIBRARIAN_SDR
+	bool "python-librarian-sdr"
+	depends on BR2_PACKAGE_PYTHON || BR2_PACKAGE_PYTHON3
+	select BR2_PACKAGE_PYTHON_LIBRARIAN
```

Comment:
Packages that have "PYTHON_LIBRARIAN" in the name, cannot select "PYTHON_LIBRARIAN" pkg. Some weirdness from buildroot.